src/ovl_3B_MariosRainbowCastle/24FAB0.c

```diff
@@ -1,6 +1,54 @@
 #include "common.h"
-
-INCLUDE_ASM(s32, "ovl_3B_MariosRainbowCastle/24FAB0", func_800F6610_24FAB0);
+#include "process.h"
+
+void SleepProcess(s32);                                  /* extern */
```

Comment:
`SleepProcess` is already declared in process.h, so you can skip the declaration here.